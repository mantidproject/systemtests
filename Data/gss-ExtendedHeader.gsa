Sample Run: 5427 Vanadium Run: UNKNOWN Wavelength: UNKNOWN
#Instrument parameter file: NOMAD_11_22_11.prm                                  
Monitor: 1
# diamond
# 6 Histograms
# File generated by Mantid:
# Instrument: NOMAD
# From workspace named : NOM_5427
# Primary flight path 19.5m 
# Sample Temperature: UNKNOWN Freq: UNKNOWN Guide: UNKNOWN
# Normalised to pCharge
# Total flight path 21.5m, tth 15deg, DIFC 1418.75
# Data for spectrum :0
BANK 1 20716 20716 SLOG 42         165308     0.0004000  0 FXYE
  41.736015418          0.004759269           0.002232955                     
  41.752709824          0.004973334           0.002282620                     
  41.769410908          0.003271038           0.001851197                     
  41.786118673          0.004823472           0.002247966                     
  41.802833120          0.011178389           0.003422153                     
  41.819554253          0.012817391           0.003664457                     
