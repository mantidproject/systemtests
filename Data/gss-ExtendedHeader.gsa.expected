ws.getNumberHistograms() == 1
ws.blocksize() == 6
